webpack/scenes/ContentViews/Details/Versions/__tests__/contentViewVersions.test.js

```diff
@@ -272,37 +329,48 @@ test('Can open Promote Modal', async (done) => {
     .post(cvPromotePath, cvPromoteParams)
     .reply(202, promoteResponseData);
 
-  const {
-    getByText, queryByText, getByLabelText, getAllByLabelText,
-  } = renderWithRedux(
+  const { getByText, queryByText } = renderWithRedux(
     withCVRoute(<ContentViewVersions cvId={5} details={cvDetailData} />),
     renderOptions,
   );
 
-  expect(queryByText(`Version ${firstVersion.version}`)).toBeNull();
-  await patientlyWaitFor(() => {
-    expect(getByText(`Version ${firstVersion.version}`)).toBeInTheDocument();
-  });
-  // Expand Row Action
-  expect(getAllByLabelText('Actions')[1]).toHaveAttribute('aria-expanded', 'false');
-  fireEvent.click(getAllByLabelText('Actions')[1]);
-  expect(getAllByLabelText('Actions')[1]).toHaveAttribute('aria-expanded', 'true');
-  fireEvent.click(getByText('Promote'));
+  // Nothing will show at first, page is loading
+  expect(queryByText('Publish new version')).toBeNull();
+  // Assert that the CV version is now showing on the screen, but wait for it to appear.
   await patientlyWaitFor(() => {
-    expect(getByText('Select a lifecycle environment from the available promotion paths to promote new version.')).toBeInTheDocument();
-    expect(getByLabelText('prod')).toBeInTheDocument();
+    expect(getByText('Publish new version')).toBeInTheDocument();
+    fireEvent.click(getByText('Publish new version'));
   });
-  // Select env prod
-  fireEvent.click(getByLabelText('prod'));
-  fireEvent.click(getByLabelText('promote_content_view'));
-  // Modal closes itself
+  await patientlyWaitFor(() => expect(getByText('Publish new version - 1.0')).toBeInTheDocument);
+
+  // useSelectorMock.mockClear();
```

Comment:
leftover, this can be removed :)

Reply:
@jeremylenz  Oh I see. This might be because I've not finished editing this file...Thanks for pointing it out :)